portability-worker/src/main/java/org/dataportabilityproject/worker/JobProcessor.java

```diff
@@ -36,61 +38,61 @@
  * (2) Run the copy job
  */
 final class JobProcessor {
-  private static final Logger logger = LoggerFactory.getLogger(JobProcessor.class);
+    private static final Logger logger = LoggerFactory.getLogger(JobProcessor.class);
 
-  private final JobStore store;
-  private final ObjectMapper objectMapper;
-  private final InMemoryDataCopier copier;
+    private final JobStore store;
+    private final ObjectMapper objectMapper;
+    private final InMemoryDataCopier copier;
 
-  @Inject
-  JobProcessor(JobStore store, ObjectMapper objectMapper, InMemoryDataCopier copier) {
-    this.store = store;
-    this.objectMapper = objectMapper;
-    this.copier = copier;
-  }
+    @Inject
+    JobProcessor(JobStore store, ObjectMapper objectMapper, InMemoryDataCopier copier) {
+        this.store = store;
```

Comment:
Looks like the formatter is switching to 4 space tabs instead of 2, could you double check that youre running the google-java-formatter?